docs/pim/enable_purchasing_products.md

```diff
@@ -45,6 +45,19 @@ ibexa:
                     - poland
 ```
 
+In the `ibexa_storefront.yaml` file, under the `ibexa.system.storefront_group.product_catalog.regions` configuration, regions are set with `default` value. Remember to either exclude this element or extend it by [configuring other regions](enable_purchasing_products.md#configuring-other-regions-and-currencies).
+
+```yaml
+ibexa:
+    system:
+        storefront_group:
+            product_catalog:
+                currencies:
+                    - EUR
+                regions:
+                    - default
```

Comment:
Maybe in the VAT rates example below, we should add the `default` region to avoid `An exception has been thrown during the rendering of a template ("Could not find 'Ibexa\Contracts\ProductCatalog\Values\RegionInterface' with identifier 'default'").`

I can't make suggestion outside PR scope so I put it here:

``` yaml
ibexa:
    repositories:
        default:
            product_catalog:
                engine: default
                regions:
                    default: ~
                    germany:
                        vat_categories:
                            standard: 19
                            reduced: 7
                            none: ~
                    uk:                                               
                        vat_categories:                            
                            standard: 20                            
                            reduced: 5                            
                            none: ~
                    poland:                                               
                        vat_categories:                            
                            standard: 23                            
                            reduced: 8                            
                            none: ~
```

Reply:
@micszo @adamwojs A reaction to this idea? I came to this because I crashed my own front (4.6.0-beta1) by copy-pasting the examples directly in config/packages/ibexa.yaml

Reply:
@adriendupuis these are great ides because then user doesn't have to take any action "ootb". Both pieces of configuration can coexist.
One thing that comes to mind, we need to consider how this could impact other default configurations, namely Nightly setup.

Reply:
@adriendupuis to double-check, do you have in mind adding such an idea to the doc? or to the default config in a clean installation?

Reply:
@micszo My goal is only to have the doc example working when its three code samples are used. The default config coming with the clean install is out of PR scope and probably should stay unchanged.